python/cudf/cudf/core/dtypes.py

```diff
@@ -78,7 +78,6 @@ def __eq__(self, other):
             return (
                 self._categories.dtype == other._categories.dtype
                 and self._categories.equals(other._categories)
-                and self.ordered == other.ordered
```

Comment:
Why are we reverting this?

Reply:
Was fixed separately here https://github.com/rapidsai/cudf/pull/5382/files